### PR TITLE
Add FASTQ info tool

### PR DESCRIPTION
**Pull request by gallardoalba**
FASTQ info is a simple tool for validating single and paired FASTQ files.

### CODE REVIEW

File: tools/fastq_info/macros.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <edam_topics>
+            <edam_topic>topic_0091</edam_topic>
+        </edam_topics>
+        <edam_operations>
+            <edam_operation>operation_0336</edam_operation>
+        </edam_operations>
+    </xml>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package">fastq_utils:@TOOL_VERSION</requirement>
```
Comment (bgruening) [inline tools/fastq_info/macros.xml:14]: ? what is that? theversion is specified in a different way ...

Comment (gallardoalba) [reply tools/fastq_info/macros.xml]: True, sorry I used the bionano macros file as template, but it used Docker.

**Comment by bgruening** [comment]
@gallardoalba did this work locally for you? I'm wondering what Galaxy installed for you with this broken requirement.

**Comment by gallardoalba** [comment]
> @gallardoalba did this work locally for you? I'm wondering what Galaxy installed for you with this broken requirement.

Yes, indeed worked :S

**Review by bgruening — APPROVED** [review]

**Pull request merged**